specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/definitions/Tasks.json

```diff
@@ -117,20 +125,20 @@
         }
       ]
     },
-    "ConnectToTargetSqlMITaskProperties": {
-      "x-ms-discriminator-value": "ConnectToTarget.AzureSqlDbMI",
+    "ConnectToSourceSqlServerSyncTaskProperties": {
+      "x-ms-discriminator-value": "ConnectToSource.SqlServer.Sync",
       "type": "object",
-      "description": "Properties for the task that validates connection to Azure SQL Database Managed Instance",
+      "description": "Properties for the task that validates connection to SQL Server and also validates source server requirements",
```

Comment:
source server requirements for continuous sync*?

just to be explicit

Reply:
Actually you have for sync scenario in connect to target, so just use that here as well.

Reply:
Done.  Also references to 'continuous' -> 'online'